src/ensembl/src/content/app/entity-viewer/components/feature-length-axis/featureLengthAxisHelper.ts

```diff
@@ -0,0 +1,58 @@
+import { ScaleLinear } from 'd3';
+
+export const getTicks = (scale: ScaleLinear<number, number>) => {
+  // use d3 scale to get 'approximately' 10 ticks (exact number not guaranteed)
+  // which are "human-readable" (i.e. are multiples of powers of 10)
+  // and are guaranteed to fall within the scale's domain
+  let ticks = scale.ticks();
+  const length = scale.domain()[1]; // get back the initial length value on which the scale is based
+
+  // choose only the "important" ticks for labelling
+  const exponent = Number(
+    (length.toExponential().match(/e\+(\d+)/) as string[])[1]
+  );
```

Comment:
Is this same as writing (String(length).length-1) ?

Reply:
I guess so...

But I found a cleaner one: `Math.floor(Math.log10(length))`